app/cosi/cosi.go

```diff
@@ -0,0 +1,143 @@
+// CoSi is a scalable protocol for collectively signing messages.
+// CoSi produces compact signatures that clients can verify efficiently,
+// and that convey the precise set of cosigners for transparency.
+package main
+
+import (
+	"os"
+	"time"
+
+	"github.com/dedis/cothority/app/lib/server"
+	"github.com/dedis/cothority/log"
+	"gopkg.in/urfave/cli.v1"
+)
+
+const (
+	// BinaryName represents the Name of the binary
+	BinaryName = "cosi"
+
+	// Version of the binary
+	Version = "0.9"
```

Comment:
well, now is a good time, as this will not be the 0.9 cosi anymore. +1 for 0.10...